### PR TITLE
Create audioWorkletStream.html

### PR DESCRIPTION
**Pull request by guest271314**
Example AudioWorklet, Transform Stream, Transferable Streams for https://github.com/microphone-stream/microphone-stream/issues/47
AudioWorklet => ReadableStream

### CODE REVIEW

File: test/resources/audioWorkletStream.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        });
+        console.log(mediaStream);
+        const worklet = URL.createObjectURL(
+          new Blob(
+            [document.querySelector('script[type=worklet]').textContent],
+            {
+              type: 'text/javascript',
+            }
+          )
+        );
```
Comment (nfriedly) [inline test/resources/audioWorkletStream.html:44]: Nice! I was hoping there would be a way to do this without creating separate files!

File: test/resources/audioWorkletStream.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        const reader = readable.getReader();
+        reader.read().then(function processAudioStream({ value, done }) {
+          if (done) return reader.closed; // Promise
+          // do stuff with value from AudioWorklet
+          console.log(
+            value instanceof Float32Array,
+            value.length,
+            value.every((float) => float === 0),
+            done
+          );
```
Comment (nfriedly) [inline test/resources/audioWorkletStream.html:57]: So, this is where we'd pass things along to the node.js-style stream, right?

Comment (guest271314) [reply test/resources/audioWorkletStream.html:57]: That is one location in the code to covert the `Float32Array`(s) from `AudioWorkletProcessor.process` to different values, for example, `Int16Array`, `Uint8Array`.

Another location is on the `readable` side which can be passed to `pipeThrough()` and `pipeTo()`, or `Response()`.  It is also possible to pass a `ReadableStream` to local or remote `quic-transport` server using `WebTransport`, where if necessary the data can be passed to node.js code.

I have little experience with node.js streams. I have experimented with Streams API https://streams.spec.whatwg.org/. If the API's are compatible, yes, that is one part of the code where the data as a `ReadableStream` can be passed to node.js.

Comment (guest271314) [reply test/resources/audioWorkletStream.html:57]: > So, this is where we'd pass things along to the node.js-style stream, right?

Yes, that is the data read in main thread from `AudioWorklet` output, logged at `console`.

**Comment by nfriedly** [comment]
@trivikr what do you think about this? Do you want to try and massage it into the current microphone-stream API?

**Comment by guest271314** [comment]
I created the file in the forks' "resources" folder. A pattern for usage, if you decide to use `AudioWorklet`. I have not read all of the code in this repository.

**Comment by nfriedly** [comment]
Actually, yea, good point. I think we can go ahead and merge this in and use it as a pattern, like you said.

**Pull request merged**